.github/workflows/pr-test-build.yml

```diff
@@ -5,34 +5,7 @@ on:
     branches: [main, staging]
 
 jobs:
+  
   build:
-    runs-on: ubuntu-latest
-    
-    steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: 'package.json'
-      - name: Test Svelte build
-        run: |
-          cd ui
-          npm i
-          npm run build
-      - name: Test Firebase API build
-        run: |
-          cd api/functions
-          npm i
-          npm run lint
-      - name: Install Latest Docker
-        run: |
-          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu  $(lsb_release -cs)  stable"
-          sudo apt-get update
-          sudo apt-get install docker-ce
-      - name: Check Docker Version
-        run: docker --version
-      - name: Test Docker build
-        run: |
-          cd docker
-          docker build -t sswconsulting/codeauditor .
\ No newline at end of file
+    uses: ./.github/workflows/test-build.yml
```

Comment:
can we prefix it with the `template-` for more readability as it is just a callable pipeline.

Reply:
All those pipelines that are only triggered by calling, I think those should be prefixed with that.